trivup/apps/SchemaRegistryApp.py

```diff
@@ -28,96 +28,115 @@
 
 from trivup import trivup
 from trivup.apps.KafkaBrokerApp import KafkaBrokerApp
+from trivup.apps.SslApp import SslApp
 
-import uuid
 import requests
-import subprocess
 
 
-class SchemaRegistryApp (trivup.App):
+sr_image = 'confluentinc/cp-schema-registry'
+
+
+class SchemaRegistryApp (trivup.DockerApp):
     """ Confluent Schema Registry app.
         Depends on KafkaBrokerApp.
         Requires docker. """
 
-    default_image = 'confluentinc/cp-schema-registry:5.2.1'
+    default_version = '5.3.0'
+
+    default_app_conf = {
+        'debug': True,
+        'host.name': 'localhost',
+        'listeners': 'http://0.0.0.0:8081'
+    }
 
-    def __init__(self, cluster, conf=None, on=None):
+    @staticmethod
+    def _conf_to_env(app_conf):
+        return ["SCHEMA_REGISTRY_{}={}"
+                .format(key.replace(".", "_").upper(), value)
+                for key, value in app_conf.items()]
```

Comment:
is this py2 and py3 compatible?

Reply:
for i in "2.7.15" "3.7.4"; do pyenv shell $i;  python -c 'import sys; print(sys.version); app_conf = {"a": "b", "c": "d"} ; print(["SCHEMA_REGISTRY_{}={}".format(key.replace(".","_").upper(), value) for key, value in app_conf.items()])'; done
pyenv: version `2.7.15' not installed
2.7.16 (default, Jul 26 2019, 10:11:46) 
[GCC 4.2.1 Compatible Apple LLVM 10.0.1 (clang-1001.0.46.4)]
['SCHEMA_REGISTRY_A=b', 'SCHEMA_REGISTRY_C=d']
3.7.4 (default, Jul 26 2019, 10:18:18) 
[Clang 10.0.1 (clang-1001.0.46.4)]
['SCHEMA_REGISTRY_A=b', 'SCHEMA_REGISTRY_C=d']